scripts/litePageSizeValidator/index.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+const { exec } = require('child_process');
+
+const litePageSizeValidator = async () => {
+  const urlsToCheck = [
+    '/hindi',
+    '/mundo/articles/cddylv9g8z0o',
+    '/nepali/bbc_nepali_radio/liveradio',
+    '/arabic/media-53135426',
+    '/marathi/popular/read',
+    '/gahuza/bbc_gahuza_radio/programmes/p0340x2m',
+    '/nepali/news-50627370',
+    '/arabic/sports-54278377',
+    '/korean/topics/cnwng7v0e54t',
+  ];
+
+  const execPromise = command => {
+    return new Promise(resolve => {
+      exec(command, (err, stdout) => {
+        resolve(stdout);
+      });
+    });
+  };
+
+  const testResults = await Promise.all(
+    urlsToCheck.map(async url => {
+      const command = `curl -sI --compressed http://localhost:7080${url}.lite?renderer_env=live | grep -i content-length | awk '{print $2/1024}'`;
```

Comment:
Because we are going to include a combination of express & nextJS pages, perhaps we should include the full url in the array e.g. `http://localhost:7080/path/to/express/page.lite?renderer_env=live` or `http://localhost:7080/path/to/nextjs/page.lite?renderer_env=live`